Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM cloudposse/terraform-root-modules:0.4.5 as terraform-root-modules
```

Comment:
let's parameterize the ENV vars like we did here
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile

Reply:
Hrmm... you mean parameterize the `0.4.5`?

Reply:
specify just a few ENV vars, and then construct the other ENV vars as we did here:
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L15
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L27
https://github.com/cloudposse/testing.cloudposse.co/blob/master/Dockerfile#L31

fewer values to manage.
I'll update it.

Reply:
Can you fix in new PR?